### ISSUE TITLE
Collapse/Expand Split.Pane with Double Click of Pane Gutter

### ISSUE THREAD

**Issue by bnapora** [gfazioli/mantine-split-pane#1]
Hi...great work on mantine-split-pane!  Very nice codebase.  

I've been testing it in our application and it is very promising.  I've looked at the documentation available, and saw the secton on "Reset with Double Click".  I'm trying to set up a 3 pane layout with a main center pane and then right and left panes that can be expanded or collapsed by clicking the pane gutter.  I've attempted to use minWdith, initialWidth, and maxWdith to make it work.  The double click collapses the pane, but does not restore to either the previous width or the initialWidth.  

Is it possible to enable double click to collapse to 0 width and then on next double click expand to initialWidth?

Not sure if you're looking at feature requests yet, but this would be useful (in my opinion.)

**Comment by gfazioli** [comment]
@bnapora Thank you! I'm glad you like the mantine-split-pane codebase.

Actually, this feature is already possible but **due to a bug** in version `0.2.2`, the double click was not being released correctly. Thanks to your report, I noticed it. 🙏

So, I have released version `0.2.3` with the double-click working and I have included an example that should address your request. [Click here](https://gfazioli.github.io/mantine-split-pane#custom-double-click) to see it in action

![Export-1712901421215](https://github.com/gfazioli/mantine-split-pane/assets/432181/4f02c919-7f4f-4318-a21d-55446e4283ce)

If you have any questions or need further assistance, feel free to ask!
